helm/prefect-server/templates/NOTES.txt

```diff
@@ -47,5 +47,14 @@
   - The location can also be changed within the UI itself per user
   - The API must be accessible from the the user's machine for the UI to work
 
+{{- if .Values.jobs.createTenant.enabled }}
+
+#4 The `{{ .Values.jobs.createTenant.tenant.name }}` tenant was created.
+   The dashboard and the Agent (if enabled) should become functional without any further action.
+
+{{- else }}
+
```

Comment:
I think we can just leave this out. The notes are already a bit lengthy as is and I want to limit it to action items.